pnpm-wireit/.changeset/chilled-rabbits-smash.md

```diff
@@ -1,5 +0,0 @@
----
-"@formidable/monorepo-tools-pnpm-wireit-dogs-core": minor
```

Comment:
Oh man!  I was wondering about these weird files.  I didn't realize they'd be automatically deleted.  Very cool!